apps/ledger-live-desktop/src/renderer/components/MainSideBar/index.tsx

```diff
@@ -237,6 +253,7 @@ const MainSideBar = () => {
   const referralProgramConfig = useFeature("referralProgramDesktopSidebar");
   const recoverFeature = useFeature("protectServicesDesktop");
   const recoverHomePath = useAccountPath(recoverFeature);
+  const ldmkTransportFlag = useFeature("ldmkTransport") as Feature_LdmkTransport;
```

Comment:
You shouldn't use type assertion here. If you rebuild your deps you will see the type of your feature flag.
As a general rule we should avoid using the `as` keyword as much as possible.
<img width="880" alt="Screenshot 2024-10-22 at 10 44 11" src="https://github.com/user-attachments/assets/5a7a868f-2739-4b1a-b263-b522eed28fcf">